### PR TITLE
Fixed grammer and spelling

### PR DESCRIPTION
**Pull request by aakajjam**
AI-262: Made some additional grammar and spelling changes

### CODE REVIEW

File: docs/High Availability, DR/Setup Global Replication.md [hunk 1]
```diff
@@ -106,10 +106,10 @@ Replace {backup-id} with the backup id that you want to restore and {service-id}
...
+      Once the restore is complete, the default username and password displayed in the "connect" window of the Secondary service will not work. Restore overwrites this information with the username and password of the Primary service. Hence, you will have to use Primary service's username and password to connect to the Secondary service.
 
 ### **Step 4: Set up Replication between the Primary and the Secondary**
-1\. Since we want to set up replication between the two SkySQL services, the Secondary service should be able to connect to the Primary service. Add the Outbound IP address of the Secondary service to the Allowlist of the Primary service. Outbound IP can be obtained from "Service Details" page in the SkySQL portal. Please add this IP to the allow list of Primary service in the portal.
+1\. Since we want to set up replication between the two SkySQL services, the Secondary service should be able to connect to the Primary service. Add the Outbound IP address of the Secondary service to the Allowlist of the Primary service. Outbound IP can be obtained from the"Service Details" page in the SkySQL portal. Please add this IP to the allowlist of Primary service in the portal.
```
Comment (vjr) [inline docs/High Availability, DR/Setup Global Replication.md:112]: There's a missing space after the "the" word added here, looks like.

**Review by vjr — APPROVED** [review]
LGTM after addressing the minor comment about a missing space after a "the" word added in this PR. Feel free to merge this PR after fixing that.

File: docs/High Availability, DR/Setup Global Replication.md (hunk 1 above)
Comment (aakajjam) [reply docs/High Availability, DR/Setup Global Replication.md]: added space with new commit.

**Comment by aakajjam** [comment]
@vjr Can you please merge this PR when you have a chance.

**Pull request merged**